php7.3/fpm-alpine/Dockerfile

```diff
@@ -39,7 +39,7 @@ RUN set -ex; \
 			| sort -u \
 			| awk 'system("[ -e /usr/local/lib/" $1 " ]") == 0 { next } { print "so:" $1 }' \
 	)"; \
-	apk add --virtual .wordpress-phpexts-rundeps $runDeps; \
+	apk add --virtual .wordpress-phpexts-rundeps $runDeps imagemagick; \
```

Comment:
We should just put it up with the other persistent dependencies; just after `ghostscript` with a similar "why" this package comment.
` # Alpine package for "imagemagick" contains ~120 .so files`

Reply:
will update - thought `--virtual .wordpress-phpexts-rundeps` might be used somewhere

Reply:
We should also include a link to this PR for posterity / details. :+1:

Reply:
applied changes and re-ran `./update.sh`